src/JoomlaBrowser.php

```diff
@@ -268,7 +268,7 @@ public function installJoomlaMultilingualSite($languages = array())
 		foreach ($languages as $language)
 		{
 			$I->debug('I mark the checkbox of the language: ' . $language);
-			$I->click(['xpath' => "//label[contains(text()[normalize-space()], '$language')]/input"]);
+			$I->click(['xpath' => "//label[contains(text()[normalize-space()], '$language')]"]);
```

Comment:
We might also have to update the function to change the Default Language from French when no language is passed.

I will send modifications.